### ISSUE TITLE
Logging issue

### ISSUE THREAD

**Issue by Knallcharge** [Dukecity/CommandSchedulerBundle#28]
Hi,
don't know if this is a bug or just a configuration issue, just thought I'd share it:
After updating to version 4, I'm facing a problem with the logging, the CommandSchedulerBundle-logging now interferes with my own (custom) logging. 
I have a custom logger for the channel "app" that logs stuff to the database, configured like this:
```
        app:
            level:    debug
            type:     service
            id:       monolog.db_handler
            channels: ["app"]
```
This worked fine with version 3 of the CommandSchedulerBundle, it logged everything to the designated logfile (cron.log), but starting with version 4, the CommandSchedulerBundle kind "grabs" my DB-logger and tries to log its stuff there (e.g. in the event "ScheduledCommandPreExecution"). Then my logger starts putting out exceptions as it needs certain data and objects for the DB, which - of course - the CommandSchedulerBundle doesn't know about.
My "fix" now was that in my DB-logger write()-method, I reject logging, when the log-attempt comes from the CommandSchedulerBundle, but of course now I don't have cron-logs at all.

So my question is: What changed in the logging-mechanism inside CommandSchedulerBundle from version 3 to 4 which might be causing this? I could imagine that this could also affect other installation without custom loggers, as CommandSchedulerBundle now uses the standard logger instead of its own (which logs to the configured file).

**Comment by Knallcharge** [comment]
Investigated some more and changed "app" to "customdb" in my monolog-config (also services.yaml) just to test this, and this makes this strange behaviour disappear, so maybe ist was just my config.

Guess I found another thing, I'll keep you updated, if I find out more (or open a PR).

**Comment by Knallcharge** [comment]
Ok, found it. Still don't understand why the bundle would "kidnap" my custom logger, but as monolog's config is quite irritating, I guess I just made a mistake somewhere that version 3 of the bundle wasn't exploiting :)

Just one thing: I found that the my logfiles (once I got them working again) were written to `/var` instead of `/var/log`, so I ended up with e.g. `/var/logscheduler.log` instead of `/var/log/scheduler.log`.

Of couse, this can be fixed easily by changing log_path in the config:
`log_path: "%kernel.logs_dir%"` to `log_path: "%kernel.logs_dir%/"`
Just thought I'd mention it as the logfiles were in the correct directory with version 3 but version 4 now expects the trailing slash in the config.
